cmd/demo/benchmark.go

```diff
@@ -53,20 +52,23 @@ func (n *node) Benchmark(args []string) error {
 	n.mtx.Lock()
 	defer n.mtx.Unlock()
 	peer := n.peers[args[0]]
-	x, _ := strconv.Atoi(args[1])
+	totalAmountEth, _ := strconv.Atoi(args[1])
```

Comment:
This is used as float below but you parse it as int here (if you change it here, you also need to use the other validator in the command definition).

Reply:
This was somewhat intentional, but allowing floats is also fine with me.

Reply:
I dont mind for now, will close it.